test/index.js

```diff
@@ -545,16 +540,16 @@ describe('mongoose-patch-history', () => {
         .catch(done)
     })
 
-    it('without changes: adds a patch', (done) => {
-      Post.update(
+    it('without changes: doesn`t add a patch', (done) => {
+      Post.updateOne(
```

Comment:
this test was the opposite. I edited it because it should not create a patch for no changes right?

Other forks have done the same.

Reply:
@vinerich

Reply:
I'll answer this more clearly with a closer look. From a semantics point of view you should be correct. I remeber something odd with this test but I'll need a little bit more time.

Reply:
@jdegger please check out #89 for a better explanation of why this test behaved as it did. When the fix is merged you can rebase i think.

However currently travis pipelines are not executing, i can't merge before.